challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js

```diff
@@ -0,0 +1,11 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+for (let i = 10; i <= 40; i = i + 2) {
```

Comment:
Variables are always defined using camelCase convention, they never start by capital letter.
You should also take in account what they are storing.
⚠️ FIX: rename properly for variables in both cases

These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.
⚠️ FIX: iterate over all numbers and print only the odd/even ones